### ISSUE TITLE
Mochi Live TV - Accessibility "features" getting enabled in app automatically (preexisting Live Channels issue)

### ISSUE THREAD

**Issue by jhedfors** [brunochanrio/MochiTIF#1]
This is a preexisting Live Channels issues with Accessibility.

When Accessibility is enabled for any other app,  accessibility "features" get enabled in this app. While the on-screen channel navigation buttons seems unnessasary, the blocking issue is with the EPG. While navigating, the EPG scrolls to the farthest point in the future for channels that contain no EPG data. This issue essentially prevents me from using any app that uses the accessibility settings (for button remapping as an example).

@brunochanrio - Could we possibly have a toggle setting to turn off this accessibility view?

**Comment by jhedfors** [comment]
Hi @brunochanrio -  I am curious if you have had a chance to look at this by enabling Accessibility for any app that request Accessibilty permissions? I really want to use an app such as Button Remapper, but am unable to as Mochi Live TV is my primary app. Thanks!

**Comment by ScriptNewbie** [comment]
I think it is coded in src/com/android/tv/ui/hideable/AutoHideScheduler.java on line 59 (at least think it is in "Live TV example app" I believe Live Channels and this app is based on). Unfortunately source code for this app is not present in this repo. 

<img width="834" alt="Zrzut ekranu 2024-01-19 o 14 44 01" src="https://github.com/brunochanrio/MochiTIF/assets/44020089/167e607d-f700-44ae-b2eb-3dcea84be126">
